### ISSUE TITLE
Modeling interpersonal relations using Bio CRM

### ISSUE THREAD

**Issue by yoge1** [InTaVia/idm-rdf#24]

**Comment by yoge1** [comment]
@ptleskin will add such relations based on Wikidata.
@sennierer and @biktorrr could you check if APIS/BiographyNet data has interpersonal relations, and if yes, model these as the above Bio CRM recipe?

**Comment by biktorrr** [comment]
So, I understand that the ones from wikidata were already added by @ptleskin, also for biographynet, right?

**Comment by ptleskin** [comment]
@biktorrr Yes, We are working on adding the interpersonal links, including all actors in InTaVia having a link to wikidata

**Comment by biktorrr** [comment]
Great. Let me know if I can shed some light on the Dutch side of things here..

________________________________
